pyramid/file_test.go

```diff
@@ -0,0 +1,134 @@
+package pyramid
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/google/uuid"
+)
+
+func TestPyramidWriteFile(t *testing.T) {
+	filename := uuid.Must(uuid.NewRandom()).String()
```

Comment:
Not a fan of UUIDs or of random data in tests, but if we're going to use them then this form is documented as _exactly_ the above:
```suggestion
	filename := uuid.New().String()
```

Reply:
Changed - need to some randomness so that subsequent runs after a fail test won't look at the same data (if failed to be deleted)